### DISCUSSION TITLE
Is there a way to break string

### DISCUSSION THREAD

**Discussion by yysg** [sgwilym/windups#60] [Q&A]
Hello, is there a way to break string when i use    `useWindupString`
such as 
`const test = "windups \n is so nice!"
 const [windup] = useWindupString(test)
 return <div>{windup}</div>
`
I hope it shows

windups
 is so nice!

**Comment by sgwilym** [comment]
useWindupString just returns a string, so newlines
should
work!

Provided you have the right `white-space` CSS styling wherever that text is being shown.
